kerrokantasi/settings/base.py

```diff
@@ -58,6 +56,15 @@ def get_git_revision_hash():
     PROTECTED_URL=(str, '/protected_media/'),
     DEFAULT_MAP_COORDINATES=(tuple, (60.192059, 24.945831)),  # Coordinates of Helsinki
     DEFAULT_MAP_ZOOM=(int, 11),
+    # Authentication settings
+    OIDC_API_AUDIENCE=(str, 'https://auth.turku.fi/kerrokantasi'),
+    OIDC_API_SCOPE_PREFIX=(str, 'kerrokantasi'),
+    OIDC_API_REQUIRE_SCOPE_FOR_AUTHENTICATION=(bool, True),
+    OIDC_API_ISSUER=(str, 'https://testitunnistamo.turku.fi/openid'),
+    OIDC_API_AUTHORIZATION_FIELD=(str, 'https://auth.turku.fi'),
+    SOCIAL_AUTH_TUNNISTAMO_KEY=(str, 'https://auth.turku.fi/kerrokantasi'),
+    SOCIAL_AUTH_TUNNISTAMO_SECRET=(str, '65be90db930a03c14fd39af6121876ceb3a5f82bc548a29781818dac'),
```

Comment:
No secrets to repositories, Might be better that we don't add Turku specific settings either.